### PR TITLE
feat: addon search

### LINKED ISSUE #43: [Features] addon search
support to search the specify addon from added indexes.

### PR DESCRIPTION
**Pull request by 1aal**
- fix #43

### CODE REVIEW

File: pkg/cmd/addon/search.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	}
+	for _, res := range results {
+		label := res.addon.Labels
+		tbl.AddRow(res.addon.Name, label[constant.AppVersionLabelKey], res.index.name)
+	}
+	tbl.Print()
+	return nil
+}
+
+func searchAddon(name string) ([]searchResult, error) {
```
Comment (ldming) [inline pkg/cmd/addon/search.go:58]: Add test case

Comment (1aal) [reply pkg/cmd/addon/search.go:58]: > Add test case

Get it!

File: pkg/cmd/addon/search_test.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package addon
```
Comment (ldming) [inline pkg/cmd/addon/search_test.go:1]: Add license header

File: pkg/cmd/addon/search.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
+package addon
```
Comment (ldming) [inline pkg/cmd/addon/search.go:1]: license header

**Review by ldming — APPROVED** [review]

**Pull request merged**